index e995857..40264ac 100644
--- a/client/tests/documentView_test.ts
+++ b/client/tests/documentView_test.ts
@@ -12,12 +12,7 @@ test("Advisory view is working", async ({ page }) => {
   await page.goto("/#/search");
   await page.getByPlaceholder("Enter a search term").fill("avendor");
   await page.getByRole("button", { name: "Search" }).click();
-  await page
-    .locator("a")
-    .filter({
-      hasText: "criticalCVSS3CVSS2SSVCSTATECVESPUBLISHERTITLETRACKING IDINITIAL RELEASECURRENT"
-    })
-    .click();
+  await page.locator('a').filter({ hasText: 'critical1CVSS3CVSS2SSVCSTATECVESPUBLISHERTITLETRACKING IDINITIAL RELEASECURRENT' }).click();
   await expect(page.getByText("Avendor-advisory-0004").first()).toBeVisible();
   await expect(page.getByText("Test CSAF document")).toBeVisible();
 });
